.evergreen/run-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       AUTH                    Set to enable authentication. Defaults to "noauth"
+#       SSL                     Set to enable SSL. Defaults to "nossl"
+#       MONGODB_URI             Set the suggested connection MONGODB_URI (including credentials and topology info)
+#       MARCH                   Machine Architecture. Defaults to lowercase uname -m
+
+
+AUTH=${AUTH:-noauth}
+SSL=${SSL:-nossl}
+MONGODB_URI=${MONGODB_URI:-}
+TESTS=${TESTS:-}
+PHP_VERSION=${PHP_VERSION:-7.2.21}
+
+# The PLATFORM environment variable is used by the matrix testing project.
+# For the time being, we can use that to install the correct extension version
+if [ "x${PLATFORM}" != "x" ]; then
+  DIR=$(dirname $0)
+  DRIVER_VERSION=1.5.5 . $DIR/install-dependencies.sh
+fi
+
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+[ -z "$MARCH" ] && MARCH=$(uname -m | tr '[:upper:]' '[:lower:]')
+
+echo "Running $AUTH tests over $SSL, connecting to $MONGODB_URI"
```

Comment:
Should we rephrase this? I imagine this might print: "Running noauth tests over nossl", which seems odd.

What are `AUTH` and `SSL` when they are set (and not defaulting to `noauth` and `nossl`, respectively)?

Perhaps something like:

* `Running tests with AUTH=$AUTH, SSL=$SSL, connecting to $MONGO_URI`

Or if `$AUTH` (and `$SSL` would only ever be) "auth" or "noauth", then perhaps:

* `Running tests with $AUTH and $SSL, connecting to: $MONGODB_URI`

Reply:
I've noticed the wording before but it has never bugged me to the point where I wanted to change it. Due to evergreen logging everything, this log line is preceded by the actual env variables a few lines above: https://evergreen.mongodb.com/task_log_raw/mongo_php_library_test_php_versions__php_versions~7.0_edge_versions~latest_stable_driver_versions~latest_stable_os_php7~debian92_test_test_replica_set_2d6fce1d04aac91fdc9d5245d5be738555921842_21_02_09_09_52_54/0?type=T#L2266, so we might as well remove this. I'll defer to your preference on this, I typically skip over this line anyways.

Reply:
If the actual environment variables are dumped above, I think we can definitely remove this.

Reply:
Since the output was due to the `xtrace` option being enabled (which I want to remove across the board for cleaner logs), I've updated the wording to your last suggestion.